From: Mikhail Titov <mlt@gmx.us>
Date: Tue, 25 Jun 2013 14:01:19 -0500
Subject: Find MPI with CMake on win32

---
 src/CMakeLists.txt | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 41f484f..378969a 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -91,6 +91,36 @@ add_executable (threshold ${THRESHOLD})
 #add_executable (compare ${OBJFILES} compare.cpp)
 #add_executable (extract ${OBJFILES} extract.cpp)
 
+find_package(MPI REQUIRED)
+if(NOT ${MPI_CXX_COMPILER})	# no mpicxx on win32
+  include_directories(${MPI_CXX_INCLUDE_PATH})
+  target_link_libraries(aread8 ${MPI_CXX_LIBRARIES})
+  target_link_libraries (areadinf ${MPI_CXX_LIBRARIES})
+  target_link_libraries (d8flowdir ${MPI_CXX_LIBRARIES})
+  target_link_libraries (d8flowpathextremeup ${MPI_CXX_LIBRARIES})
+  target_link_libraries (d8hdisttostrm ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinfavalanche ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinfconclimaccum ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinfdecayaccum ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinfdistdown ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinfdistup ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinfflowdir ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinfrevaccum ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinftranslimaccum ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dinfupdependence ${MPI_CXX_LIBRARIES})
+  target_link_libraries (dropanalysis ${MPI_CXX_LIBRARIES})
+  target_link_libraries (gridnet ${MPI_CXX_LIBRARIES})
+  target_link_libraries (lengtharea ${MPI_CXX_LIBRARIES})
+  target_link_libraries (moveoutletstostrm ${MPI_CXX_LIBRARIES})
+  target_link_libraries (peukerdouglas ${MPI_CXX_LIBRARIES})
+  target_link_libraries (pitremove ${MPI_CXX_LIBRARIES})
+  target_link_libraries (slopearea ${MPI_CXX_LIBRARIES})
+  target_link_libraries (slopearearatio ${MPI_CXX_LIBRARIES})
+  target_link_libraries (slopeavedown ${MPI_CXX_LIBRARIES})
+  target_link_libraries (streamnet ${MPI_CXX_LIBRARIES})
+  target_link_libraries (threshold ${MPI_CXX_LIBRARIES})
+endif(NOT ${MPI_CXX_COMPILER})
+
 install(TARGETS aread8 
                 areadinf
                 d8flowdir
